types/projects.ts

```diff
@@ -0,0 +1,51 @@
+import { Build, SubbuildStatus } from '@hubspot/local-dev-lib/types/Build';
```

Comment:
Nit, but in LDL we capitalize the filenames for everything in the `types/` folder